CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Prowide Core - CHANGELOG
 
+#### 10.3.0 - SNAPSHOT
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)